projects/1.21/assets/ars-nouveau/ars_nouveau/lang/zh_cn.json

```
@@ -0,0 +1,2009 @@
+  "ars_nouveau.page1.armor_upgrading": "魔法盔甲可使用纤维升级，以获得额外效果。每种盔甲的纤维槽数目不同，纤维槽的等级也不同。槽位的等级越高，纤维的能力就越强，部分纤维更是要求有最低等级需求。纤维的编织方法请参阅改衣台条目。",
+  "ars_nouveau.page1.basic_spell_turret": "给予炮塔红石信号可让其施法，和发射器类似。炮塔接受使用“触摸”或“弹射物”的法术。使用抄有法术的法术羊皮纸可设置法术。炮塔的施法需要从附近魔源罐抽取魔源。若炮塔有容器相邻，则其可使用“拾取”和“放置方块”。",
+  "ars_nouveau.page1.better_casting": "你的魔力池可通过特殊魔法盔甲、魔咒、习得新魔符、饮用药水等方法扩充。获得法师之花种子之后，即可合成许多种可自由配置的魔法盔甲，再使用纤维加以调整。",
+  "ars_nouveau.page1.bookwyrm_charm": "书龙护符可对存储讲台使用，以增加其可用容器上限。以书与笔增强启蒙仪式即可获得护符。书龙可以染成白色、黑色、蓝色、绿色、红色、紫色。",
+  "ars_nouveau.page1.decorative": "纯粹的装饰性方块。可向切石机放入魔源石以查看完整列表。",
+  "ars_nouveau.page1.deny_scroll": "可为自动化相关实体提供物品列表。将其放置抄写台上，再潜行对其使用物品或方块，即可将该物品或方块抄入卷轴。将卷轴交予星宝石兽，它就只会捡起或拿取不在卷轴中的物品。",
+  "ars_nouveau.page1.drygmy_charm": "德格米通常会跟随和照料其周围的动物。它们会出现在世界各处，虽然不算常见。可以在世界中为德格米指定家园，它们会产出附近怪物和动物的击杀掉落物，而不会实际伤害它们。可以在野生的德格米附近丢下荒野之角以表示友好！德格米可以染成青色、橙色、棕色。",
+  "ars_nouveau.page1.enchanters_fishing_rod": "可在抄写台上向巫师钓鱼竿抄入单个法术。钓鱼竿的法术会以“触摸”起始，抄入的法术不可具有形态。巫师钓鱼竿可像普通的钓鱼竿一样使用；但在勾住实体后，持有者还可向该实体施法，直至收回渔线。",
+  "ars_nouveau.page1.enchanters_gauntlet": "可在抄写台上向巫师护手抄入单个法术。护手的法术会以“触摸”起始，抄入的法术不可具有形态。使用护手施放法术可降低消耗。",
+  "ars_nouveau.page1.enchanting_apparatus": "附魔装置会使用基座和魔源进行合成。使用附魔装置时，应在距其3格内的任意奥术基座处放置所需物品。准备妥当之后，应对附魔装置使用配方中央的物品。需要在附魔装置基底的相邻处放置奥术核心，否则它无法运作。",
+  "ars_nouveau.page1.flask_cannons": "烧杯加农炮会消耗物品栏中药水瓶和药水烧瓶内的药水，并据此发射出喷溅药水或滞留药水。发射前，应先在其圆盘菜单内选择使用何种药水。",
+  "ars_nouveau.page1.how_to_enchant": "附魔装置可使用魔源和物品添加或升级魔咒。开始时，应先选定一个I级魔咒，并将相应物品放入基座。再在附近放置一罐魔源，并对附魔装置使用希望附魔的物品。附魔装置只会附上与需附魔的物品兼容的魔咒。",
+  "ars_nouveau.page1.illusion_blocks": "对镜像编织块使用方块后，其会复刻该方块的外观。这些方块的碰撞和光照特性与所复刻的方块一致。“感知魔法”能够揭示幻象。所有种类的幻象方块均可用作传送门框架。对幻象方块使用“构筑魔光”，可让其发光。",
+  "ars_nouveau.page1.imbuement_chamber": "向物品注入魔源，以制造新物品。灌注室是获取魔源宝石的主要途径，可以使用紫水晶和青金石制造魔源宝石。灌注室会从环境中常态积累魔源进行加工，也会从2格内的魔源罐抽取魔源。部分配方要求在距灌注室1格内的基座上放置附加物品，例如制造精华的配方。基座中的物品不会被消耗。",
+  "ars_nouveau.page1.item_detector": "展示盒可在容器内某物达到一定条件时输出红石信号。对展示盒使用物品，即可设置为探测该物品。对方块交互可增加阈值，敲击可降低。先对容器使用支配之杖，再对展示盒使用，即可建立连接。潜行对其使用支配之杖可将触发条件在小于和大于阈值间切换。",
+  "ars_nouveau.page1.magelight_torch": "装饰性光源。对烛台使用“构筑魔光”可点燃烛台。火焰的颜色与法术颜色一致。与墙上的魔光火把交互，可更改其火焰的方向。烛台需要“触摸”或“弹射物”接“敏感”才能选定。",
+  "ars_nouveau.page1.mimic_scroll": "可为自动化相关实体提供物品列表。将此物品置于附在容器上时，实体即只会向容器存入其中已有的物品。在容器上放置物品展示框，再将模仿卷轴放入，即可为容器附加卷轴。",
+  "ars_nouveau.page1.mob_jar": "可用于捕捉和存储生物，以供运输或装饰。捕捉生物需施展收容仪式。更多信息请参阅收容仪式条目。对罐施放“解除魔法”即可将罐中生物释放到罐上方。罐上方的音符盒会播放罐内生物的声音。",
+  "ars_nouveau.page1.new_glyphs": "获得新法术构件前，需要预先备好相关设备、资源，还需搭建部分基地设施。习得新法术构件需借助魔符。魔符可在抄写台处使用经验和物品制作。获得之后，使用魔符即可完成记忆。注解手抄本可以用来在玩家之间分享魔符。更多信息请参阅抄写台相关条目。",
+  "ars_nouveau.page1.obtaining_gems": "为获得魔源宝石，首先需要一个灌注室。灌注室会向内部的物品灌注魔法，并将其转变成新物品。向灌注室放入紫水晶或青金石，再等待片刻即可获得魔源宝石。灌注室会消耗附近魔源罐中的魔源来加速合成。在游戏早期，可以使用占卜杖来寻找紫水晶母岩。",
+  "ars_nouveau.page1.performing_rituals": "仪式的施展需要仪式火盆和仪式板。准备好之后，先将火盆放置出来，再对其使用仪式板。此操作会点燃火盆，并令其进入等待激活状态。此状态下，可向火盆投入额外物品进行增强，简单向火盆顶部扔出物品即可。",
+  "ars_nouveau.page1.potion_flask": "最多能存储8份药水的烧瓶。对药水罐使用烧瓶，或在工作台内将烧瓶与药水合成，均可填充烧瓶。潜行对药水罐使用烧瓶可清空烧瓶。",
+  "ars_nouveau.page1.reactive_enchantment": "拥有反应魔咒的物品有概率在挥动时施法。魔咒会使用配方中法术羊皮纸上的法术。",

```

```suggestion
  "ars_nouveau.page1.reactive_enchantment": "拥有反应魔咒的物品有概率在挥动时施法。魔咒会使用法术羊皮纸抄写在物品上的法术。",

```
The spell on the Spell Parchment determines the spell that will be inscribed on the item. 在法术羊皮纸上的法术决定了会被刻在物品上的法术。
感觉这里的译法出入太大了，因此给出结合两种译法的建议。

---

换成“魔咒会使用抄写所用的法术羊皮纸上的法术”
这里的右边一页刚好是附魔装置加反应魔咒的配方，里面就用到了法术羊皮纸